zyte_common_items/util.py

```diff
@@ -100,3 +100,21 @@ def __init__(self, target):
 
     def __call__(self, value):
         return cast_metadata(value, self._target)
+
+
+def wrap_description_into_html(description: str) -> str:
```

Comment:
I think this function is specific to a particular field; it might be better to move it to processors.py or some other location, not add it to the utils.py